types/hand-crafted/JitsiTranscriptionStatus.d.ts

```diff
@@ -1,4 +1,4 @@
-export const enum JitsiTranscriptionStatus {
+export enum JitsiTranscriptionStatus {
```

Comment:
Why converting a `const enum` to an `enum`?

The difference between them is explained here: https://tusksoft.com/blog/march-2015/const-enums-in-typescript-1-4-and-how-they-differ

Reply:
We want to use the auto generated types (which are omitted by using const enums) to access them outside of this lib ("see comment above"). More on this can be found [here](https://ncjamieson.com/dont-export-const-enums/).